### PR TITLE
Challenge2 day2

### PR DESCRIPTION
**Pull request by AineKiraboMbabazi**
**Lists and strings , loops** 
Count vowels in a string, every letter in the english alphabet is either a vowel or consonant, [a, e, i, o, u] is a list of all vowels. The rest are all considered consonants. Write a function that takes a string and returns a tuple containing a new string made of all and only the vowels from the original string and the number of duplicates in the original string. Only the first instance of the vowel is considered. 
**For example:** 
countVowels(‘dahdah’)  # will return (‘a’, 3) countVowels(‘drink water’) # will return (‘iae’, 0) 
**Python programming challenge : Fizzbuzz, (lists and conditional logical)** 
Write a function named Fizzbuzz that takes two(2) lists and returns Fizz if the combined length of the lists is divisible by 3, Buzz if is divisible by 5, Fizzbuzz if it is divisible by both 5 and 3  or the combined length of the list. 
**For example:** 
fizzbuzz([1, 2, 3], [ ])    # will return “fizz” fizzbuzz([1, 2, 3], [1, 2])  # will return “buzz” fizzbuzz([1, 2, 3], [1]) # will return 4

### CODE REVIEW

**Comment by AineKiraboMbabazi** [comment]
@dennisja

File: test_countvowels.py [hunk 1]
```diff
@@ -2,12 +2,15 @@
 import unittest
 class testcountvowels(unittest.TestCase):
```
Comment (dennisja) [inline test_countvowels.py:3]: Cahnege the name of the class

File: countvowels.py [hunk 2]
```diff
@@ -1,22 +1,24 @@
 def vowel_count(txt):
+    if not isinstance (txt,str):
+        raise TypeError ('invalid input')
+    else:
```
Comment (dennisja) [inline countvowels.py:4]: Redundant else

**Review by dennisja — CHANGES_REQUESTED** [review]

File: fizzbuzz.py [hunk 3]
```diff
@@ -1,33 +1,25 @@
...
     else:
-        message=len(newlist)
-    return message
+        newlist=a+b
+        mylist=len(newlist)
+        if(mylist%5==0 and mylist%3==0):
+            message="fizzbuzz"
+            
+        
+        elif (mylist%5==0):
```
Comment (dennisja) [inline fizzbuzz.py:11]: You can fore gore using the `elif` statements by just returning the messages

**Pull request merged**